lib/code_climate.ex

```diff
@@ -5,7 +5,24 @@ defmodule CodeClimate do
 
   @config_filename "/config.json"
   @cmd "mix"
-  @default_opts ~w[credo --format=codeclimate]
+  @default_opts ~w[credo --format=json]
+
+  @issue_category %{
+    "Credo.Check.Design.DuplicatedCode" => ["Duplication"],
```

Comment:
Might be worth extracting the category names to constants? e.g. `@category_duplication "Duplication"`. Up to you.